config.yaml

```diff
@@ -0,0 +1,51 @@
+kube:
+  config: out-of-cluster
+  context: minikube
+  job:
+    active.deadline.seconds: 60
+    retries: 0
+  service.account.name: default
+  wait.for.resource.poll.count: 5
+  log.process.wait.time: 60
+log.level: debug
+app.port: 5000
+default.namespace: default
+redis:
+  address: localhost:6379
+  max.active.connections: 10
+logs.stream:
+  read.buffer.size: 140
+  write.buffer.size: 4096
+postgres:
+  user: postgres
+  password:
+  host: localhost
+  port: 5432
+  database: proctord_test
+  max.connections: 50
+  connections.max.lifetime: 30
+new.relic:
```

Comment:
`new.relic` or `new_relic`? Since the former might be confusing in determining the namespace of the config.

Reply:
It legacy code will break deployment if changed.